src/backend/src/modules/pia-intake/metadata/pia-status.metadata.ts

```diff
@@ -283,8 +283,63 @@ export const piaStatusMetadata: Partial<
           },
         ],
       },
+      [PiaIntakeStatusEnum.PENDING_COMPLETION]: {
+        allow: true,
+      },
+    },
+  },
+  [PiaIntakeStatusEnum.PENDING_COMPLETION]: {
+    updates: {
+      allow: false,
+    },
+    transition: {
+      [PiaIntakeStatusEnum.INCOMPLETE]: {
+        allow: true,
+        conditions: [
+          {
+            accessType: [UserTypesEnum.CPO],
+          },
+        ],
```

Comment:
On transitioning from PENDING_COMPLETION to Edit statuses, review needs to be deleted.

```suggestion
        ],
        actions: [
          {
            type: 'update',
            key: 'review',
            value: null,
          },
        ],
```